.github/workflows/pre-release-continuous-test.yaml

```diff
@@ -10,14 +10,21 @@ on:
     branches:
       - pre-release-continuous-test
   schedule:
-    # Run hourly
-    - cron: "0 * * * *"
+    # Run every 3 hours
```

Comment:
I would prefer making it 6 hours so that we can have 4-5 hours long running forge for nightly :)

Reply:
Changed to 6 hrs